examples/01-Getting-started.ipynb

```diff
@@ -7,7 +7,7 @@
    "metadata": {},
```

Comment:
<p>We could add a bit of description here around how to use synthetic data instead </p><pre class="ql-syntax" spellcheck="false">data = generate_data(“movielens-1m", num_rows=10000)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/313/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
I would not add synthetic data in the Getting Started notebook. This should be a really easy example. MovieLens 1M takes <1min to download and extract.

Reply:
I added @marcromeyn suggestion to the 2nd example notebook, and kept Getting started as it is to keep it clean. In particular bcs generate_data also accepts schema instead of dataset name, and that concept is only introduced in the 2nd example.